### DISCUSSION TITLE
Is this project complete?

### DISCUSSION THREAD

**Discussion by tydog98** [scarburato/t150_driver#31] [General]
Is this driver complete or is development just inactive?

**Comment by scarburato** [comment]
the latter lol

**Comment by albfan** [comment]
I think basic features are covered these days (all report comes from device internals, so any change by hardware (change range using mode+ left/right arrow, flip clutch and accel with press 2 seconds mode) just works

We still missing some config features. For that: @scarburato is there a guide to analyze usb captures on windows? I just found some discussion here: https://github.com/Kimplul/hid-tmff2/issues/3
